Jenkinsfile

```diff
@@ -87,36 +87,37 @@ pipeline {
                 sh 'mvn -U -B -e clean install -DskipTests -T1C'
             }
         }
-
```

Comment:
we should remove this commit from the history

Reply:
I beg to differ : with sequential stages the build takes over 7hours 
with parallel stages 
- stable tests take about 5h30 
- unstable tests take 2h10
build failure in 52 cannot be attributed to running parallel stages : the test which failed is a webmin integration test which is the last module to run in the stable tests. By then the unstable test stage had been completed for over 3hours...
I'm pretty sure the issue is with the Linshare singleton wait strategy: it looks like some containers start before containers they depend on are fully started

Reply:
I ended up dropping the parallel stage for now this is an optimization and it is more unstable than unoptimized builds.
optimization can come in a second step